.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
What's the difference between this bug report template and the one called [`ISSUE_TEMPLATE.md`](https://github.com/xbmc/Kore/pull/588/files#diff-1a1c3dd142f76a5fad803a0c52839881)